usr/share/rear/conf/default.conf

```diff
@@ -805,6 +805,20 @@ CLONE_ALL_USERS_GROUPS="no"
 # into config file /etc/rear/local.conf
 SSH_ROOT_PASSWORD=
 
+# SSH_PRIVATE_KEYS_RECOVER_PASSPHRASE defines a passphrase for SSH private keys during 'rear recover'.
+# If set, ReaR will set the passphrase for each SSH private key which was not passphrase-protected on the original
+# system, before copying the private key to the rescue medium.
+# An empty passphrase (the default) implies that unprotected private keys will not be copied to the rescue medium.
+# CAUTION: Configuration files are copied to the rescue medium. If you define a verbatim passphrase in a configuration
+# file, anyone with physical access to the rescue medium would be allowed unauthenticated access to each account on the
+# network authorizing one of those private keys.
+# Recommendation: Set the passphrase by reading an access-protected file, which will not be copied to the rescue medium.
+# E.g., create
+#   echo "my_recover_passphrase" > /root/.ssh/rear-recover-passphrase && chmod u=rw,go=- /root/.ssh/rear-recover-passphrase
+# then use
+#   SSH_PRIVATE_KEYS_RECOVER_PASSPHRASE="$(cat /root/.ssh/rear-recover-passphrase 2>/dev/null)"
```

Comment:
`$(< file)` is more concise and avoids the external program call. If you want to use cat then please don't toss the error output so that it can show up in the log.